File: Java-Solutions/Problem6.java
//Problem 6 - sum square difference

import java.lang.Math;

public class Problem6 {
    public static int sumOfSquares(int n) {
        int sumOfSquaresVar = 0;

        for (int i=1; i<=n;i++) {
            sumOfSquaresVar+=Math.pow(i,2);
        }

        return sumOfSquaresVar;
    }

    public static double squareOfSums(int n) {
        int squareOfSumsVar = 0;

        for (int i=1; i<=n;i++) {
            squareOfSumsVar+=i;
        }

        double squareOfSumsVarDouble = squareOfSumsVar;

        return Math.pow(squareOfSumsVarDouble, 2.0);
    }

    public static void main(String[] args) {
        System.out.printf("%.1f", squareOfSums(100) - sumOfSquares(100));
    }
}

File: Python-Solutions/Problem6.py
# Sum Square Difference 

def sumOfSquares(n):
    sumOfSquaresVar=0
    for i in range(1,n+1):
        sumOfSquaresVar+=i**2
    return sumOfSquaresVar

def squareOfSums(n):
    squareOfSumsVar = 0
    for i in range(1,n+1):
        squareOfSumsVar+=i
    
    return squareOfSumsVar**2

print(squareOfSums(100) - sumOfSquares(100))